avocado/core/test.py

```diff
@@ -148,14 +148,14 @@ def record_and_warn(*args, **kwargs):
         self.whiteboard = ''
 
         self.running = False
-        self.time_start = None
-        self.time_end = None
+        self.time_start = 0
```

Comment:
`_tag_start()` and `_tag_end()` set proper (`time.time()`) values here, so I think that for better consistency with the test result values, which actually from the test state which in turn come from those attributes, to use `-1` here as defaults.